### PR TITLE
Update dutch.xml to 7.7.2

### PR DESCRIPTION
**Pull request by xylographe**
Based on {english,german}.xml v7.6.5

### CODE REVIEW

File: PowerEditor/installer/nativeLang/dutch.xml [hunk 1]
```diff
@@ -265,13 +271,13 @@
 					<Item id="45002" name="Unix-indeling"/>
 					<Item id="45003" name="Mac-indeling"/>
 					<Item id="45004" name="ANSI"/>
-					<Item id="45005" name="UTF-8"/>
+					<Item id="45005" name="UTF-8-BOM"/>
```
Comment (MAPJe71) [inline PowerEditor/installer/nativeLang/dutch.xml]: `<Item id="45005" name="UTF-8 zonder BOM"/>`
`<Item id="45005" name="UTF-8 met BOM"/>`

File: PowerEditor/installer/nativeLang/dutch.xml [hunk 2]
```diff
@@ -407,21 +419,44 @@
...
 				<Item id="2"    name="Sluiten"/>
 			</MD5FromTextDlg>
 
+			<SHA256FromFilesDlg title="SHA-256 code voor bestanden genereren">
+				<Item id="1922" name="Bestanden selecteren..."/>
+				<Item id="1924" name="Naar klembord kopiëren"/>
+				<Item id="2"    name="Sluiten"/>
+			</SHA256FromFilesDlg>
+
+			<SHA256FromTextDlg title="Generate SHA-256 digest">
```
Comment (MAPJe71) [inline PowerEditor/installer/nativeLang/dutch.xml]: `<SHA256FromTextDlg title="SHA-256 code genereren">`

File: PowerEditor/installer/nativeLang/dutch.xml [hunk 3]
```diff
@@ -407,21 +419,44 @@
 				<Item id="1903" name="Naam van programma, map of document:"/>
 				<Item id="1"    name="OK"/>
 				<Item id="2"    name="Annuleren"/>
-				<Item id="1904" name="Opslaan..."/>
+				<Item id="1904" name="Opslaan"/>
```
[the remark this answers is not in the document]
Comment (schnurlos) [reply PowerEditor/installer/nativeLang/dutch.xml]: Just a comment from my side: In that case i use the "…" sign (three periods, but just one sign)

**Comment by xylographe** [comment]
Thank you, Menno, for your comments. The errors should be fixed now (fingers crossed).

Regarding "UTF-8" and "UTF-8-BOM", these are the terms used in the status bar, and also in `{english,german}.xml`. Unless there is an urgent reason to deviate in `dutch.xml`, I would suggest to keep them consistent. The explanatory addition in "UTF-8 (zonder BOM)" was not added by me, it has been there for a very long time, and as it is not incorrect, I believe, it might be best to keep the existing translation.

Regarding "Openen in standaardviewer" vs "In standaardviewer openen", in Dutch and German (cf. "Im Standard-Betrachter öffnen" in `german.xml`) it is customary to have the verb at the end — "Opdrachtprompt openen", "Bestandslocatie openen", "Laatste bestand opnieuw openen", etc.

File: PowerEditor/installer/nativeLang/dutch.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (MAPJe71) [reply PowerEditor/installer/nativeLang/dutch.xml]: Yes, you could use the ellipsis character but a set of three periods is more commonly used in menus and the period is ready available on the keyboard.

**Comment by MAPJe71** [comment]
I only use the `english.xml` file to find out what has to be added, removed and changed in the `dutch.xml` file. I'm Dutch and don't look at the German translations.

Most of the other entries have the verb at the end so yes it's better to use "In standardviewer openen".

**Comment by xylographe** [comment]
New (force-pushed) commit adds a few typo corrections (‘Kaptialen’, ‘wijzingen’, ‘projekt’), and one change:

In the Settings > Preferences dialogue, in the tab selection box on the left, the Dutch text ‘Recent geopende bestanden’ is too wide to fit into the control. Fortunately, this text is defined in a separate XML attribute. The new text is ‘Recente bestanden’, the caption on the right side, and the corresponding menu item in the File menu are not affected.

Voorkeuren

**Comment by xylographe** [comment]
New (force-pushed) commit adds new **DoSaveOrNot** tag (commit 3f7956dc1bd16445f031a38bca8aa9e72806a408).

**Comment by xylographe** [comment]
New (force-pushed) commit adds correction in Find/Replace dialogue:

The **Alt-A** keyboard accelerator was used in two buttons, "Alle vervangen" and "Vervangen in alle geopende bestanden" in the Replace tab. After correction **Alt-B** will be used for  "Vervangen in alle geopende bestanden" (Replace tab) and "In alle geopende bestanden zoeken" (Search tab).

**Comment by xylographe** [comment]
New (force-pushed) commit fixes and adds keyboard accelerators in main menu.

**Comment by xylographe** [comment]
New (force-pushed) commit fixes original problem reported in [issue #5587](https://github.com/notepad-plus-plus/notepad-plus-plus/issues/5587#issue-437530909) by replacing keyboard accelerator **Alt-A** in Mark tab of Find dialogue with **Alt-M** — "**A**lle zoeken" → "Alle **m**arkeren".

**Comment by xylographe** [comment]
New (force-pushed) commit adds new **ExitToUpdatePlugins** & **NeedToRestartToLoadPlugins** tags (commit eed0c620bbe6bf1048e8d7a45b6468465ef359de).

**Comment by xylographe** [comment]
New (force-pushed) commit adds new **AsciiInsertion** tags (commit 6a30c31e0a8dbfaeababa461e3c66564e5c6cf17).

**Comment by xylographe** [comment]
New (force-pushed) commit adds new Dialog.Find tags (commit 6f3166c86526137d8487e7bdae5627f8081bb0a8).
Note: I have chosen `"▼ Volgende"`, because `"▼ Volgende zoeken"` is approximately three pixels too wide to fit in the button control decently.

**Comment by xylographe** [comment]
New (force-pushed) commit adds new **summary** tags (commit 418d03acfe707599bbdcfae068f3e8c179a7408e).

**Comment by donho** [comment]
Thank you for the update.
Sorry, I didn't check if your PR contain all the new added entries. Could you check the file in the following issue :
#5962
and confirm me your PR contain all the new added entries, please?
If not, please add another commit to include all of them.
Thank you.

**Comment by xylographe** [comment]
New (force-pushed) commit removes **summary-filelen** tag (commit e50236f559a8e4bbd9c243046c7c66f631025ecd),
and updates **Native-Langue "version"** to 7.7.2.

@donho
All the new added entries are included.

**Pull request closed without merging**